shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -378,6 +378,68 @@ - (BOOL)isEqualTo:(FlutterTextRange*)other {
 }
 @end
 
+#pragma mark - FlutterTokenizer
+
+@implementation FlutterTokenizer {
+  FlutterTextInputView* _textInputView;
```

Comment:
Please use a property so we can know if you intend this to be a `strong` or a `weak` reference.  Right now this is being treated as a weak reference.  I suspect you want a strong reference.

Reply:
This should be a weak reference, FlutterTextInputView owns the FlutterTokenizer. I will update